apps/newsletters-ui/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<title>Newsletters Launch Wizard’</title>
+		<title>Newsletters tool</title>
```

Comment:
Should it be "Newsletters tool" or "Newsletters Tool" ?
![image](https://user-images.githubusercontent.com/74301289/235608602-d87e1bd2-cd74-43aa-98c9-e06b4b0e1c85.png)

Reply:
😆  🤷‍♂️